services/account/docs/endpoints.md

```diff
@@ -81,7 +81,7 @@ The DSNP Gateway Account Services Prototype API is designed for managing DSNP id
 - **Description**: Account still pending creation
 
 - **Status Code**: `200 OK`
-- **Schema**: [AccountResponse](#AccountResponse)
+- **Schema**: [AccountResponseDto](#AccountResponseDto)
```

Comment:
Should this also be changed or not?

Perhaps this doc should be deleted now?

Reply:
Yes, this file (also endpoints.json) is obsolete. All the API is now generated from the source code.